Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Search/SearchWidget.cpp

```diff
@@ -159,6 +164,27 @@ namespace AzToolsFramework
                 m_filter->AddFilter(FilterConstType(m_typesFilter));
                 SetTypeFilters(buildTypesFilterList());
             }
+
+            auto sourceFilter = new EntryTypeFilter();
+            sourceFilter->SetName("Source");
+            sourceFilter->SetEntryType(AssetBrowserEntry::AssetEntryType::Source);
+            m_projectSourceFilter->AddFilter(FilterConstType(sourceFilter));
+
+            auto pathFilter = new AssetPathFilter();
+            pathFilter->SetAssetPath(AZStd::string_view{ AZ::Utils::GetProjectPath() });
+            m_projectSourceFilter->AddFilter(FilterConstType(pathFilter));
+        }
+
+        void SearchWidget::FilterProjectSourceAssets()
```

Comment:
nit: This function reads like it sometimes removes the filter instead of performing a filter. Would a name like `ToggleProjectSourceAssetsFilter` be correct here, or is removing the filter considered a type of filtering?